app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -79,6 +86,60 @@ class RevisionCardFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  private fun setUpRevisionNavigationCards(
+    binding: RevisionCardFragmentBinding,
+    topicId: String,
+    subtopicId: Int,
+    subtopicListSize: Int
+  ) {
+
```

Comment:
Nit: remove extra newline for style consistency.

Reply:
done